pkg/k8sops/statefulset.go

```diff
@@ -254,3 +257,72 @@ func (k *K8sops) BuildM3DBCoordinatorPorts() []v1.ContainerPort {
 		},
 	}
 }
+
+func (k *K8sops) EnsureStatefulSets(cluster *myspec.M3DBCluster) (map[string]string, error) {
+	placementDetails := make(map[string]string) // key is pod name, value zone
+
+	// ensure zones are present in spec
+	if len(cluster.Spec.Zones) != 0 {
+		// TODO(PS) add zone distribution algo to range
+		// loop through zones
+		for zoneIndex, zoneReplicaCount := range []int32{1, 1, 1} {
```

Comment:
is this going to be driven off the `cluster` object eventually?

Reply:
yes, just updated the PR with some simple logic